site/docs/flink.md

```diff
@@ -97,15 +97,20 @@ CREATE CATALOG hive_catalog WITH (
   'catalog-type'='hive',
   'uri'='thrift://localhost:9083',
   'clients'='5',
-  'property-version'='1'
+  'property-version'='1',
+  'warehouse'='hdfs://nn:8020/warehouse/path'
 );
 ```
 
+We must set `hive-conf-dir` property to get the custom warehouse location by specifying the hive configuration directory where to load the `hive-site.xml`, if we don't provide the `warehouse` property explicitly.
```

Comment:
The document doesn't look clear enough. Actually, there are three options:
-Configure URI and warehouse
-Configure hive-conf-dir
-The resource of class path contains hive conf

Users can choose one of them